index.html

```diff
@@ -0,0 +1,135 @@
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+            </ul>
+            <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+            <div class="platform">
+                <button class="pla__btn">html</button>
+                <button class="pla__btn">css</button>
+                <button class="pla__btn">javascript</button>
+            </div>
+            <button class="see-project">See Project</button>
+        </section>
+
+        <section class="covers">
+            <img class="img__class" src="./img/pic2.png" alt="">
+            <h1 class="bold-text">Multi-Post Stories</h1>
+            <ul class="selections">
+                <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>CANOPY</li>
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+            </ul>
+            <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+            <div class="platform">
+                <button class="pla__btn">html</button>
+                <button class="pla__btn">css</button>
+                <button class="pla__btn">javascript</button>
+            </div>
+            <button class="see-project">See Project</button>
+        </section>
+
+        <section class="covers">
+            <img class="img__class" src="./img/pic3.png" alt="">
+            <h1 class="bold-text">Tonic</h1>
+            <ul class="selections">
+                <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>CANOPY</li>
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
+                <li class="circle__wrapper"><a href="#" class="circle"></a></li>
+                <li class="list__wrapper"><a href="#" class="list">2015</a></li>
+            </ul>
+            <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
+            <div class="platform">
+                <button class="pla__btn">html</button>
+                <button class="pla__btn">css</button>
+                <button class="pla__btn">javascript</button>
+            </div>
+            <button class="see-project">See Project</button>
+        </section>
```

Comment:
- [ ] [OPTIONAL] For the project's card I think it's better to use `div` tag instead of the `section` tag. This way you'll have a number of divs wrapped in a section. It will make your webpage well structured.